presage/src/manager/registered.rs

```diff
@@ -392,7 +392,7 @@ impl<S: Store> Manager<S, Registered> {
                 .as_secs();
 
             if let Some(expiration) = sender_certificate.and_then(|s| s.expiration().ok()) {
-                expiration >= seconds_since_epoch - 600
+                seconds_since_epoch <= expiration.epoch_millis() + 600
```

Comment:
I think you will need to convert the `epoch_millis` to seconds here.

Reply:
Good catch. Thank you.